R/clean_data.R

```diff
@@ -1,232 +1,197 @@
+#'      arguments that determine how constant rows and columns will be handled.
+#'      They are the input values for the `remove_duplicates()` function.
+#'   \item `standardize_date`: A list of parameters that will be used to
+#'      standardize the date values from the input data. They represent the
+#'      input values for the `standardize_dates()` function.
+#'   \item `standardize_subject_ids`: A list of parameters that are needed to
+#'      check the IDs that comply with the expect format. These arguments are
+#'      the input values of the `check_subject_ids()`.
+#'   \item `to_numeric`: a vector of column names. When provided, the values in
+#'      these columns will be converted into numeric.
+#'   \item `dictionary`: A data frame that will be used to substitute the
+#'      current values in the specified columns the those in the dictionary. It
+#'      is the main argument for the `clean_using_dictionary()` function.
 #'   }
 #'
-#' @return a list of the following 2 elements:
+#' @return A list of the following 2 elements:
 #'  \enumerate{
-#'    \item `data`: the cleaned data frame according to the user-specified
-#'          parameters
-#'    \item `report`: an object of type list with the details from every
-#'          cleaning operation.
+#'    \item `data`: A  clean data frame  according to the user-specified
+#'          parameters.
+#'    \item `report`: A list with  details from each
+#'          cleaning operation considered.
 #'  }
+#'
 #' @export
 #'
-#' @details
-#' If `check_timeframe = TRUE` and `timeframe = NULL`, the timeframe will be
-#' today's date and the same date 50 years before.
+#' @examples
+#' keep = NULL #: column names standardization will be performed across
+#' # all columns
 #'
-#' in `clean_data()`, duplicated rows will be identified across the
-#' user-specified or all columns. Once detected, all occurrences of the
-#' duplicated rows will be removed except the first. If you only need to find
-#' and remove specific duplicates, use the `find_duplicates()` then
-#' `remove_duplicates()` functions.
+#' # Parameters for substituting missing values with NA:
+#' replace_missing_values = list(target_columns = NULL, na_strings = "-99")
+#'
+#' # Parameters for duplicates removal across all columns
+#' # (target_columns = NULL)
+#' remove_duplicates = list(target_columns   = NULL,
+#'                          rm_empty_rows    = TRUE,
+#'                          rm_empty_cols    = TRUE,
+#'                          rm_constant_cols = TRUE)
+#'
+#' # Parameters for dates standardization
+#' standardize_date = list(target_columns  = NULL,
+#'                         error_tolerance = 0.5,
+#'                         format          = NULL,
+#'                         timeframe       = as.Date(c("1973-05-29",
+#'                                                     "2023-05-29")))
+#'
+#' # Parameters for subject IDs standardization
+#' standardize_subject_ids = list(id_col_name,
```

Comment:
`id_col_name = "study_id"`
I think that's what was missing

Reply:
Thanks - it works perfectly after correcting this